db/migrate/20200109111923_add_refuse_to_give_feedback_to_reference.rb

```diff
@@ -0,0 +1,5 @@
+class AddRefuseToGiveFeedbackToReference < ActiveRecord::Migration[6.0]
+  def change
+    add_column :references, :refuse_to_give_feedback, :boolean
```

Comment:
Yes, it might be a good idea.
Just, the card did mention anything about:
- Didn't respond in time
- Email failed
also, sometimes you need to combine the various states:
when `Feedback provided`
`consent_to_be_contacted` can be true or false

It does sounds a refactor when will implement the cases **Email failed**
and **Didn't respond in time**

Reply:
Sorry, my initial comment wasn't very clear!

I've added a `feedback_status` in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1077. We should add a `feedback_refused` status to the enum and use that instead of the boolean.